pandas/core/indexes/base.py

```diff
@@ -1493,13 +1493,14 @@ def _mpl_repr(self):
     # introspection
     @property
     def is_monotonic(self):
-        """ alias for is_monotonic_increasing (deprecated) """
+        """ Alias for is_monotonic_increasing.
+        .. deprecated :: """
```

Comment:
Besides leaving the quotes in their own lines, do you mind adding a blank line between the sentences